cn_indian_payroll/cn_indian_payroll/overrides/salary_slip.py

```
@@ -1909,42 +1895,41 @@ def tax_calculation(self):
+                        percentage.append(first_slab_percentage)
+                        difference.append(first_slab_value)
+                        total_value.append(first_slab_amount)
                     self.custom_tax_slab = []
                     for i in range(len(from_amount)):
                             self.append("custom_tax_slab", {
                             "from_amount": from_amount[i],
                             "to_amount": to_amount[i], 
-                            "percentage":  percentage[i]   ,
+                            "percentage":  percentage[i],
                             "tax_amount":total_value[i],
                             "amount":difference[i]     
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)